Maple2.Model/Game/User/Account.cs

```diff
@@ -9,6 +9,7 @@ public class Account {
     public long Id { get; init; }
 
     public required string Username { get; init; }
+    public required string Password { get; init; }
```

Comment:
Do we need Password on this Account? I would figure it's only used on DB Model. Once a user has logged in, it's no longer necessary.

Reply:
its in use inside GlobalService